File: coupons-backend-php/api/domain/Promotion.php
<?php
class Promotion {
    public int $id_promotion;
    public int $id_coupon;
    public int $percentage;
    public string $start_date; // 'Y-m-d' formato de fecha
    public string $end_date; // 'Y-m-d' formato de fecha
    public bool $is_enabled;

    public function __construct(
        int $id_promotion, 
        int $id_coupon, 
        int $percentage, 
        string $start_date, 
        string $end_date, 
        bool $is_enabled
    ) {
        $this->id_promotion = $id_promotion;
        $this->id_coupon = $id_coupon;
        $this->percentage = $percentage;
        $this->start_date = $start_date;
        $this->end_date = $end_date;
        $this->is_enabled = $is_enabled;
    }
}
?>

File: CouponsClientBackendC#/CouponsClient.BC/Models/Promotion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CouponsClient.BC.Models
{
    public class Promotion
    {
        public int IdPromotion { get; set; }
        public int IdCoupon { get; set; }
        public Coupon Coupon { get; set; }
        public int Percentage { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public bool IsEnabled { get; set; }
    }

}
